### DISCUSSION TITLE
GPS updates to live map

### DISCUSSION THREAD

**Discussion by rpratt20** [dl9rdz/rdz_ttgo_sonde#430] [Q&A]
I have a puzzling situation with gps position. I am using a repurposed DFM17 as a gps connected to pins 12, 13 of a Lilygo board. When I depart my fixed location the gps position updates 1 time at the 200 meter radius as programmed. This position then does not update until return to home and the fixed station position is again used at approximately 100 meters from home. At any time during travel I can cut power and following startup the correct gps position is updated 1 single time but will not move. However on return home the fixed location again snaps into place.  When I enable Sondehub updates, the chasecar position will randomly update to correct positions on Sondehub but remain fixed on the livemap screen.   It appears the program always knows the correct position but updates to livemap are frozen except for the transition between fixed and mobile station.

**Comment by dl9rdz** [comment]
Ok... this is strange and not as it should be. I am currently traveling and will be back to a TTGO with GPS only in 2 weeks from now so I can't check this myself right now.

Just to confirm, so you have a fixed position configured in the TTGO config (Config/General/Receiver fixed lat/lon)?

Can you check this URL: http://<your ttgo ip>/live.json ?
This should have a "gps": part at the end, like
`"gps": {"lat": 64.1214, "lon": -21.8963, "alt": 20, "sat": 0, "speed": 0, "dir": 0, "hdop": 0 }}`
Would like to know (a) if this part is there, and (b) if the position in there is updated (also sat, hdop should not be 0 if GPS is working - its 0 for the fixed position)

**Comment by rpratt20** [comment]
This is parked at home after restarting network and web pages. Livemap and Sondehub in agreement. Phone and program on home network.{"sonde": {"type":"DFM","id": "","ser": "","frame": 0,"vframe": 0,"time": 0,"sats": 0,"freq": 405.01,"rssi": 0,"afc": 0,"launchKT": 0,"burstKT": 0,"countKT": 0,"crefKT": 0,"launchsite": "Rapid City","res": 1 }, "gps": {"lat": 43.7107, "lon": -98.0083, "alt": 396, "sat": 0, "speed": 0, "dir": 0, "hdop": 0 }}The gps portion did not change over night.Let me list settings before I forget.Chasemapper 1Sondehub reporting 1Sondehub location reporting 2Sondehub frequency import 0I will connect to iPhone hotspot and drive next.Just prior to entering chase mode and just after I do not get any live.jason page. Both Livemap shows correct heading and speed data 12 hdop 0.7 and Sondehub updated to new position. Turning west now. Stop about 90m west at next corner. Live map no change but Sondehub is good after a few seconds delay. No live.jason. Turn north. Drive 400 meters about. Livemap no update. Sondehub now at 4 minutes no update. But updated as I was typing. Will reboot to RDZ access point to look for live.jason.After reboot Livemap has correct position with sats=3 and hdop=25.5.  Live.jason remains blank. Returning toward starting point. Stop short over home location. No update to Livemap or Sondehub. Suspect no internet connection when using access point explains Sondehub. Continue route and Livemap position jumps from reboot position to fixed location. Sondehub no update. Live. Jason still no update. The page appears hung up as when I hold a finger on it I have spinning icon.RandySent from my iPhoneOn Mar 13, 2024, at 18:51, dl9rdz ***@***.***> wrote:﻿
Ok... this is strange and not as it should be. I am currently traveling and will be back to a TTGO with GPS only in 2 weeks from now so I can't check this myself right now.
Just to confirm, so you have a fixed position configured in the TTGO config (Config/General/Receiver fixed lat/lon)?
Can you check this URL: http:///live.json ?
This should have a "gps": part at the end, like
"gps": {"lat": 64.1214, "lon": -21.8963, "alt": 20, "sat": 0, "speed": 0, "dir": 0, "hdop": 0 }}
Would like to know (a) if this part is there, and (b) if the position in there is updated (also sat, hdop should not be 0 if GPS is working - its 0 for the fixed position)

**Comment by rpratt20** [comment]
Maybe you figured out this before I did. I was spelling json incorrectly. I have several friends named Jason. Anyway, I repeated the drive with live.json. Not only did spell check not capitalize Jason but I got data that matched the display updates. So yes in fact the json stream to live map only updates when switching from or to fixed location or during restart.RandySent from my iPhoneOn Mar 14, 2024, at 11:19, Randall Pratt ***@***.***> wrote:﻿This is parked at home after restarting network and web pages. Livemap and Sondehub in agreement. Phone and program on home network.{"sonde": {"type":"DFM","id": "","ser": "","frame": 0,"vframe": 0,"time": 0,"sats": 0,"freq": 405.01,"rssi": 0,"afc": 0,"launchKT": 0,"burstKT": 0,"countKT": 0,"crefKT": 0,"launchsite": "Rapid City","res": 1 }, "gps": {"lat": 43.7107, "lon": -98.0083, "alt": 396, "sat": 0, "speed": 0, "dir": 0, "hdop": 0 }}The gps portion did not change over night.Let me list settings before I forget.Chasemapper 1Sondehub reporting 1Sondehub location reporting 2Sondehub frequency import 0I will connect to iPhone hotspot and drive next.Just prior to entering chase mode and just after I do not get any live.jason page. Both Livemap shows correct heading and speed data 12 hdop 0.7 and Sondehub updated to new position. Turning west now. Stop about 90m west at next corner. Live map no change but Sondehub is good after a few seconds delay. No live.jason. Turn north. Drive 400 meters about. Livemap no update. Sondehub now at 4 minutes no update. But updated as I was typing. Will reboot to RDZ access point to look for live.jason.After reboot Livemap has correct position with sats=3 and hdop=25.5.  Live.jason remains blank. Returning toward starting point. Stop short over home location. No update to Livemap or Sondehub. Suspect no internet connection when using access point explains Sondehub. Continue route and Livemap position jumps from reboot position to fixed location. Sondehub no update. Live. Jason still no update. The page appears hung up as when I hold a finger on it I have spinning icon.RandySent from my iPhoneOn Mar 13, 2024, at 18:51, dl9rdz ***@***.***> wrote:﻿
Ok... this is strange and not as it should be. I am currently traveling and will be back to a TTGO with GPS only in 2 weeks from now so I can't check this myself right now.
Just to confirm, so you have a fixed position configured in the TTGO config (Config/General/Receiver fixed lat/lon)?
Can you check this URL: http:///live.json ?
This should have a "gps": part at the end, like
"gps": {"lat": 64.1214, "lon": -21.8963, "alt": 20, "sat": 0, "speed": 0, "dir": 0, "hdop": 0 }}
Would like to know (a) if this part is there, and (b) if the position in there is updated (also sat, hdop should not be 0 if GPS is working - its 0 for the fixed position)
